sites/fast-website/src/app/components/content-placement-container/content-placement-container.template.ts

```diff
@@ -20,23 +25,31 @@ export const ContentPlacementContainerTemplate = html<ContentPlacementContainer>
             <site-card-section>
                 ${repeat(
                     x => x.featureCardData,
-                    html`<site-feature-card>
-                        <div>${x => x.item}</div>
-                        <h4>${x => x.header}</h4>
-                        <p slot="body">${x => x.body}</p>
-                        <fast-anchor
-                            slot="footer"
-                            href=${x => x.githubLink}
-                            appearance="lightweight"
-                            >View Github</fast-anchor
-                        >
-                        <fast-anchor
-                            slot="footer"
-                            href=${x => x.documentationLink}
-                            appearance="lightweight"
-                            >Read Documentation</fast-anchor
-                        >
-                    </site-feature-card>`
+                    html`
+                        <site-feature-card>
+                            <div>
+                                ${(x, c) =>
+                                    c.index.toString.length === 1
```

Comment:
Should this be `toString()`?

Reply:
Yes. Thank you for catching this. John's comment below is to remove the index and use css. So this code will be removed.